Sources/com.elgato.philips-hue.sdPlugin/pi/js/pi.js

```diff
@@ -54,6 +54,14 @@ function PI(inContext, inLanguage, inStreamDeckVersion, inPluginVersion) {
             document.getElementById('groups').label = instance.localization['GroupsTitle'];
         }
     };
+    
+    // Save IP addresses for bridges into global settings
+    this.saveCachedBridges = function(cache) {
+        Object.keys(cache).forEach(function(id) {
+            globalSettings.bridges[id].ip = cache[id].ip;
```

Comment:
This will override the cached ip.
- The discovery process might find a new ip for the same id (e.g. DHCP use-case) and we'll want to update the cache with the new one